pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -64,6 +64,7 @@
   private static final String METRICS_CONFIG_PREFIX = "pinot.broker.metrics";
   private static final long DEFAULT_BROKER_DELAY_SHUTDOWN_TIME_MS = 10 * 1000L;
   private static final String BROKER_DELAY_SHUTDOWN_TIME_CONFIG = "pinot.broker.delayShutdownTimeMs";
+  private static final String PINOT_BROKER_TABLE_LEVEL_METRICS = "pinot.broker.tablelevel.metrics";
```

Comment:
I don't really like this name, tbqh. It's not really clear what it means to have a "tablelevel" metric. It would be clearer to have something like `pinot.broker.perTableMetrics = true/false` or `enablePerTableMetrics=true/false`